### PR TITLE
Cloud SIEM Release Notes for April 11, 2024

### PR DESCRIPTION
**Pull request by pkazmir-sumo**
Cloud SIEM Release Notes for April 11, 2024

Includes MITRE Explorer enhancements and UI Light Theme

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-cse/2024-04-11-application-update.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: April 11, 2024 - Application Update
+keywords:
+  - cloud siem
+  - MITRE
+  - UI theme
```
Comment (kimsauce) [inline blog-cse/2024-04-11-application-update.md:6]:
```suggestion
  - mitre
  - light theme
```

File: blog-cse/2024-04-11-application-update.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+### MITRE ATT&amp;CK&reg; Explorer Enhancements
+
+We're excited to announce multiple enhancements to our MITRE ATT&amp;CK Threat Coverage Explorer.
+
+* **Rules Filtering** - You can now easily filter the coverage visualization based on rules, including rules you've created and out-of-the-box rules, as well as enabled, disabled, production and prototype rules.
```
Comment (kimsauce) [inline blog-cse/2024-04-11-application-update.md:21]:
```suggestion
* **Rules Filtering** - You can now easily filter the coverage visualization based on rules, including out-of-the-box and user-created rules, as well as enabled, disabled, production and prototype rules. 
```

File: blog-cse/2024-04-11-application-update.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+The theme is set per user, and can be changed on the Sumo Logic user preferences page:
+
+<img src={useBaseUrl('img/release-notes/cse/theme-setting.jpg')} alt="Option to change UI theme"/>
+
+Note that the setting currently only affects Cloud SIEM and the Automation Service, but in the future this setting will also affect other pages in the Sumo Logic UI.
+
+### Bug fixes
+
+* Sometimes Terraform would time out waiting for match lists to be updated.
```
Comment (kimsauce) [inline blog-cse/2024-04-11-application-update.md:43]:
```suggestion
* Terraform no longer times out while waiting for match lists to be updated.
```

File: blog-cse/2024-04-11-application-update.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+image: https://help.sumologic.com/img/sumo-square.png
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+### MITRE ATT&amp;CK&reg; Explorer Enhancements
```
Comment (jpipkin1) [inline blog-cse/2024-04-11-application-update.md:18]:
```suggestion
### MITRE ATT&amp;CK&reg; Coverage Enhancements
```

Comment (jpipkin1) [reply blog-cse/2024-04-11-application-update.md:18]: It's "MITRE ATT&CK Coverage" in the UI. We should align with that in the release notes.

File: blog-cse/2024-04-11-application-update.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+We're excited to announce multiple enhancements to our MITRE ATT&amp;CK Threat Coverage Explorer.
```
Comment (jpipkin1) [inline blog-cse/2024-04-11-application-update.md:20]:
```suggestion
We're excited to announce multiple enhancements to our MITRE ATT&amp;CK Coverage page.
```

File: blog-cse/2024-04-11-application-update.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+* **Rules Filtering** - You can now easily filter the coverage visualization based on rules, including out-of-the-box and user-created rules, as well as enabled, disabled, production and prototype rules. 
+* **All Community Activity** - This view now defaults to show only the vendor and product logs that are being sent to Cloud SIEM from your data sources. This gives you a better comparison between what your theoretical and historical coverage shows and what other customers of Cloud SIEM using those same log sources are seeing. You can still change the filter to display other (or all) log sources.
+* **Customizable Colors** - You can now customize the tile colors to your own scheme.
+
+<img src={useBaseUrl('img/release-notes/cse/mitre-attack-explorer-custom-colors.jpg')} alt="Custom MITRE ATT&amp;CK Explorer Color Palette"/>
```
Comment (jpipkin1) [inline blog-cse/2024-04-11-application-update.md:26]:
```suggestion
* **Customizable Colors** - You can now customize the tile colors to your own scheme. <br/><img src={useBaseUrl('img/release-notes/cse/mitre-attack-explorer-custom-colors.jpg')} alt="Custom MITRE ATT&amp;CK Explorer Color Palette"/>
```

File: blog-cse/2024-04-11-application-update.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+For full details, see the [MITRE ATT&amp;CK Explorer documentation](/docs/cse/administration/mitre-coverage/).
```
Comment (jpipkin1) [inline blog-cse/2024-04-11-application-update.md:28]:
```suggestion
For full details, see the [MITRE ATT&amp;CK Coverage documentation](/docs/cse/administration/mitre-coverage/).
```

**Review by kimsauce — COMMENTED** [review]
I swapped out your Preferences page image with a high-res version. Original was small and a bit pixelated.

File: blog-cse/2024-04-11-application-update.md (hunk 5 above)
Comment (pkazmir-sumo) [reply blog-cse/2024-04-11-application-update.md:20]: We should leave this one.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**